src/test_l1_dsFPD.c

```diff
@@ -75,19 +75,15 @@
 #include <ut.h>
 #include <ut_log.h>
 #include "dsFPD.h"
-#include "frontPanelSettings.hpp"
 #include "stdbool.h"
 #include <ut_kvp_profile.h>
 static int gTestGroup = 1;
 static int gTestID = 1;
 
-
 /* Global flags to support features */
 static bool extendedEnumsSupported=false; //Default to not supported
-
-#define INDICATOR_COLORS_SIZE_1 (sizeof(kIndicatorColors_1)/sizeof(dsFPDColorConfig_t))
-#define INDICATOR_COLORS_SIZE_2 (sizeof(kIndicatorColors_2)/sizeof(dsFPDColorConfig_t))
-#define INDICATOR_COLORS_SIZE_3 (sizeof(kIndicatorColors_3)/sizeof(dsFPDColorConfig_t))
+					  //
+#define DS_FPD_KEY_SIZE 128 
 
 #define INDICATOR_COLORS_SIZE INDICATOR_COLORS_SIZE_1
```

Comment:
Where is this INDICATOR_COLORS_SIZE_1 define now?

Reply:
Done